app/models/transaction.rb

```diff
@@ -1,16 +1,22 @@
+require 'transaction_update_service'
+
 class Transaction < ActiveRecord::Base
   validates_presence_of :from, :to, :amount
-  before_save :send_transaction
+  before_create :send_transaction
```

Comment:
This is good, may want to check other models to see if anything else is still using before_save (or whatever else) when it'd be ideal to be more specific.